modules/ROOT/pages/import/file-provision.adoc

```diff
@@ -2,23 +2,29 @@
 = File provision
 :description: This section describes how to provide files for import.
 
+You start by connecting your data source, which can be a SQL database or local flat files.
+
 In essence, you provide a set of files that contain the data to be imported and Import imports them into your instance.
 
-Import supports flat files, i.e. files that contain data in a tabular format where each row represents a record and each column represents a field in that record.
-The most common format for such files is CSV (comma-separated values), but Import also supports TSV (tab-separated values).
+Import supports SQL and flat files, i.e. files that contain data in a tabular format where each row represents a record and each column represents a field in that record.
+The most common format for flat files is CSV (comma-separated values), but Import also supports TSV (tab-separated values).
+
+When connecting to a remote data source, the files are provided for you from that database.
+
+But when you stream your local CSV files, the process can be more iterative and manual.
 
-Import requires all files to have a header row and at least one row of data.
+Import requires all CSV files to have a header row and at least one row of data.
 The header row speficies how the data in the file should be interpreted and contains information for each field.
 For more information about CSV files and the header format, see link:{neo4j-docs-base-uri}/operations-manual/current/tools/neo4j-admin/neo4j-admin-import/#import-tool-header-format[Operations Manual -> CSV header format].
 Keep in mind that the column names must be unique, i.e. it is not possible to have two columns with the same name within the same file.
 
-The files are provided in the _Files_ panel of Import.
+The CSV files are provided in the _Files_ panel of Import.
 You can browse for them or drag and drop them into the panel.
 Once a file is added to the panel, you can preview the header and the first row of data in the file by expanding the file.
 
 [.shadow]
 image::files.png[]
 
-When you provide files to Import, they are temporarily stored on the client side until you run the import.
+When you provide CSV files to Import, they are temporarily stored on the client side until you run the import.
```

Comment:
to be more accurate, you could say something like:

 a reference to the files on your local filesystems is kept and used to send the necessary commands to stream the data to your target database on running the import. The files themselves are not uploaded to a server.
 
 This can then segue to the next point about needing to re-provide if the page is reloaded.

Reply:
I rewrote that a little for readability, but yes, good point!